src/frontends/onnx/tests/tests_python/test_backend.py

```diff
@@ -415,10 +415,6 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
         "OnnxBackendNodeModelTest.test_bitwise_or_ui64_bcast_3v1d_cpu",
         "OnnxBackendNodeModelTest.test_bitwise_xor_ui64_bcast_3v1d_cpu",
     ),
-    (
-        xfail_issue_99949,
-        "OnnxBackendNodeModelTest.test_bitwise_not_3d_cpu",
-    ),
```

Comment:
Hi,  @p-wysocki could you please help review this onnx part change? Thx

Reply:
Done :)